charts/cray-metallb/Chart.yaml

```diff
@@ -23,23 +23,23 @@
 #
 apiVersion: v2
 name: cray-metallb
-version: 1.2.2
+version: 1.3.0
```

Comment:
This change seems to me to be worthy of incrementing that the major version to `2.0.0`.